crud/rooms.py

```
@@ -49,7 +49,7 @@ async def create_room(room: rooms_schema.RoomCreate):
     else: return {"result": "Error"}
```

If you have an error you should raise an Exception with clear message

---

ok